var/spack/repos/builtin/packages/trilinos/package.py

```diff
@@ -514,6 +525,12 @@ def setup_dependent_package(self, module, dependent_spec):
     def setup_build_environment(self, env):
         spec = self.spec
         if "+cuda" in spec and "+wrapper" in spec:
+            # Add more debug options for CUDA
+            if (
+                spec.variants["build_type"].value == "RelWithDebInfo"
+                or spec.variants["build_type"].value == "Debug"
+            ):
```

Comment:
FWIW. Could simplify:

```suggestion
            if spec.variants["build_type"].value in ["RelWithDebInfo", "Debug"]:
```

Reply:
Excellent.